graveler/ref/resolver_test.go

```diff
@@ -43,10 +43,11 @@ func TestRefManager_Dereference(t *testing.T) {
 		t.Fatalf("unexpected error: %v", err)
 	}
 
-	ids := make([]graveler.CommitID, 0)
 	for iter.Next() {
 		c := iter.Value()
-		ids = append(ids, c.CommitID)
```

Comment:
I don't understand.  Where do we validate that we actually received good values, or even the correct number of values  E.g. if the iterator never iterates and ends directly, this passes.  (Not introduced in this PR, of course, so fine to open an issue and assign to whoever should fix)

Reply:
Adding a test code to compare the values.
I think it was made just to check there was no error after scanning the commit log.